pkgs/shells/oh-my-zsh/default.nix

```diff
@@ -0,0 +1,71 @@
+# This script was inspired by the ArchLinux User Repository package:
+#
+#   https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=oh-my-zsh-git
+
+
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  name = "oh-my-zsh-git-${version}";
+  version = "2016-02-27";
+
+  src = fetchgit {
+    url = "https://github.com/robbyrussell/oh-my-zsh";
+    rev = "bd6dbd1d9b1fc8a523aaf588492eb3ed4113b49d";
+    sha256 = "0bn3jijxjhrd00mc3biqs7jj6in3ivhr6d02mp4566i2rdp9x2d5";
+  };
+
+  phases = "installPhase";
+
+  installPhase = ''
+  outdir=$out/share/oh-my-zsh
+  template=templates/zshrc.zsh-template
+
+  mkdir -p $outdir
+  cp -r $src/* $outdir
+  cd $outdir
+
+  rm MIT-LICENSE.txt
+  rm -rf .git*
+
+  chmod -R +w templates
+
+  # Change the path to oh-my-zsh dir and disable auto-updating.
+  sed -i -e "2c\\ZSH=$outdir/" \
+         -e 's/\# \(DISABLE_AUTO_UPDATE="true"\)/\1/' \
+   $template
+
+  # Look for .zsh_variables, .zsh_aliases, and .zsh_funcs, and source
+  # them, if found.
+  cat >> $template <<- EOF
+
+  # Load the variables.
+  if [ -f ~/.zsh_variables ]; then
+      . ~/.zsh_variables
+  fi
+
+  # Load the functions.
+  if [ -f ~/.zsh_funcs ]; then
+    . ~/.zsh_funcs
+  fi
+
+  # Load the aliases.
+  if [ -f ~/.zsh_aliases ]; then
+      . ~/.zsh_aliases
+  fi
+  EOF
+
+  echo "Oh My Zsh has been installed into $out/share/oh-my-zsh."
+  echo
+  echo "To use Oh My Zsh, run the following command:"
+  echo "  $ cp $outdir/$template ~/.zshrc"
```

Comment:
If I remove the installation instructions from here, is it a good idea to put them into the meta-attribute `longDescription`?